doc/bt/ZC_SESSION_OBJ_ADD.bt

```diff
@@ -0,0 +1,20 @@
+//------------------------------------------------
+//--- 010 Editor v8.0 Binary Template
+//
+//      File: ZC_SESSION_OBJ_ADD.bt
+//   Authors: celophi
+//   Version: i171140
+//   Purpose: Adds a single session object.
+//  Category: 
+// File Mask: 
+//  ID Bytes: 
+//   History: 
+//------------------------------------------------
+
```

Comment:
No properties? The size is dynamic, so I'd assume there might be some, since there are no strings. I also have some old logs of this packet, and it's a bit larger there. Did you log this, or is it based on the client?
```text
EB0CFFFFFFFF2400D1BF0B002070EAAAC9E70100CCC40000000000000000900000000000
8B0CFFFFFFFF2400FC030000A879AD01DF990000E91500000000000000000000FC030000
E00CFFFFFFFF2400A086010019000000DE8A0100FFA10000000000000000000000000000
```

Reply:
I added this template earlier, and I believe your version is wrong.